www/_posts/2020-03-18-wkwebviewonly.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+author:
+    name: Niklas Merz
+    url: https://twitter.com/niklasmaerz
+title:  "How To Use 'WKWebViewOnly'"
+categories: howto
+tags: ios
+---
+
+Apple started to reject apps with UIWebView references. The next major version of cordova-ios will contain no usage of UIWebView. You might need to update your app to fix this before the next version is released and this post sums up the required steps.
```

Comment:
```suggestion
Apple started to reject apps with `UIWebView` references. You will need to update your app to fix and this post sums up the required steps.
```

Reply:
I moved the information about next major in its own section at the bottom. `Future Roadmap`